src/api/app/models/package.rb

```diff
@@ -929,8 +929,8 @@ def buildresult(prj = project, show_all = false)
   end
 
   def jobhistory_list(project, repository, arch, options = {})
-    options[:limit] = 100 if options[:limit].blank?
-    options[:package] = name if options[:package].blank?
+    options[:limit] |= 100
+    options[:package] |= name
```

Comment:
This does not work when it is an empty string, does it?

Reply:
is `package[:name]` is `""` it is imposible that you have the instance package from that name... :no_mouth:

Reply:
This should be ``||=``

Reply:
Maybe we could even drop the limit as it is nowhere used currently...

Reply:
IIRC, we added `limit` to be able to change it in the future. However, if the limit in the `backend` by default is 100 we could drop it